src/components/touchableOpacity/index.tsx

```diff
@@ -44,8 +42,10 @@ export type TouchableOpacityProps = Omit<RNTouchableOpacityProps, 'style' | 'onP
      */
     customValue?: any;
     style?: StyleProp<ViewStyle> | Animated.AnimatedProps<StyleProp<ViewStyle>>;
-    onPress?: (props: TouchableOpacityProps) => void;
-  };
+    onPress?: () => void;
```

Comment:
Why did u change the signature of onPress? 
Is that relevant for the fix? 
It just we have a custom signature for onPress because we pass the props back to the user

Reply:
No, not required, but I am pretty much sure that TouchableOpacity doesn't take any arguments when used onPress.
And even in UILib, we are not passing anything as an argument in onPress.

Reply:
But actually, React Native Touchable Opacity accepts 1 argument as GestureResponderEvent, but what you had passed was not GestureResponderEvent but rather complete GestureResponderEvent.

Let me update the pull request with GestureResponderEvent

Reply:
But it's not suppose to be an event. 
If you'll look at the Button component implementation, you'l see we pass back the prop to the user. 
This is helpful with avoiding passing inline callbacks to `onPress`

Reply:
Oh, my bad, I thought it might be an event.
Okay, let me update the code again.